app/serializers/api/v1/order_shallow_serializer.rb

```diff
@@ -0,0 +1,62 @@
+module Api::V1
+  class OrderShallowSerializer < ApplicationSerializer
+    embed :ids, include: true
+    attributes :status, :created_at, :code, :detail_type, :id, :detail_id,
+      :contact_id, :local_order_id, :organisation_id, :description, :activity,
+      :country_name, :state, :purpose_description, :created_by_id,
+      :gc_organisation_id, :processed_at, :processed_by_id, :cancelled_at, :cancelled_by_id,
+      :process_completed_at, :process_completed_by_id, :closed_at, :closed_by_id, :dispatch_started_at,
+      :dispatch_started_by_id, :submitted_at, :submitted_by_id, :people_helped, :beneficiary_id
+
+    def local_order_id
+      (object.detail_type == "LocalOrder" || object.detail_type == "StockitLocalOrder") ? object.detail_id : nil
```

Comment:
How about using `["LocalOrder",  "StockitLocalOrder"].include?(object.detail_type )  ? object.detail_id : nil`.

Just a thought.

CC: @steveyken @Darshan-Chauhan @abulsayyad123

Reply:
A bit better looking. We can make cosmetic changes later since this code has not been modified in this PR and we want to roll this out ASAP for testing